qiita_pet/handlers/analysis_handlers.py

```diff
@@ -75,7 +76,7 @@ def post(self):
         name = self.get_argument('name')
         desc = self.get_argument('description')
         analysis = Analysis.create(self.current_user, name, desc,
-                                   from_default=True)
+                                   qiita_config.portal, from_default=True)
```

Comment:
It is already part of the config object. But I agree, it will be extremely useful to fail at the moment that Qiita starts if the portal type does not exist. That can be the last step of the config reading. This way, even if the configuration of the DB is wrong, it will not allow the system to start.